### DISCUSSION TITLE
Upcoming C-x g binding in Emacs 28

### DISCUSSION THREAD

**Discussion by basil-conto** [magit/magit#4311] [General]
This is just a heads-up that the development version of Emacs 28 is currently binding `C-x g` to `revert-buffer` globally: https://git.savannah.gnu.org/cgit/emacs.git/commit/?id=7d15fa008af774789a91d7242055dc87497df66f

This arose from the following discussion: https://debbugs.gnu.org/46151

**Comment by tarsius** [comment]
Thanks for the information.

Initial reaction: that's to bad, I won't change anything.

**Reply by tarsius** [reply]
Turns out that because we try so hard to not step on the users toe we do have to do something to maintain the status quo: https://github.com/magit/magit/commit/75ae42a5bb8a764c648be2af58c5843469dd8cf1.

Also worth a look: #4237.

Please note how hard we try not to step on the users toes.

**Reply by basil-conto** [reply]
`revert-buffer` has now been moved to `C-x x g` under the new `ctl-x-x-map` (ignore the typo in the commit summary): https://git.savannah.gnu.org/cgit/emacs.git/commit/?id=a6a5d6a27a86396ab96662fa158cdcc854bd777b

**Reply by tarsius** [reply]
Thanks for the follow up.

Reverted kludge in https://github.com/magit/magit/commit/09e0ee66ccf777159cc88588e1e8b47c1abebdf3.

**Comment by basil-conto** [comment]
As expected, there's bound to be some more discussion on this: https://lists.gnu.org/r/emacs-devel/2021-02/msg00036.html
